docs/csharp/write-safe-efficient-code.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Write safe and efficient code
```

Comment:
maybe add a suffix here like - C# to help with SEO? Or maybe Write safe and efficient C# code?